Feature/add protocol 4#11

Braak-1991 opened the pull request:
Added the log from last week (29.04.2020). Open for Review

nailend commented on protokolle/29-04-2020_protokoll.md:
```diff
@@ -0,0 +1,262 @@
+
+
+# Aufgaben
+☐ ☑ ☒
+   ☐ Metadaten zu den CSV-Dateien "HTW-Wetter" und "Sonja-Messdaten" generieren [ALL]
+   ☐ die richtigen Datenbankbefehle für die Fehler -> Lücken finden und Lücken schließen [ALL]
```
add additional lines inbetween

khelfen replied:
Write as following:

    - [ ]  Text
    - [x] Text 2

to create Checkboxes. It will look like this:

- [ ] Text
- [x] Text 2

nailend commented on protokolle/29-04-2020_protokoll.md:
```diff
@@ -0,0 +1,262 @@
+   
+
+
+	* Fragen zu Git:
```
make a  headline with ##

nailend commented on protokolle/29-04-2020_protokoll.md:
```diff
@@ -0,0 +1,262 @@
+
+## Feedback zur Übung / Generell:
+
+    * Relativ wenig Antworten  
+
+    * Werden die nächsten Tage bewertet  
```
remove indentation before *

nailend commented on protokolle/29-04-2020_protokoll.md:
```diff
@@ -0,0 +1,262 @@
+
+## Fragen zu git und GitHub:
+    
+    *Wie kann ich einen Pull Request über git scm in den Master starten?* Hab das jetzt über GitHub gemacht. [Link](https://github.com/htw-pv3/literature/pull/9) 
+	
+    ->Die Pull Requests (PR) sollten über GitHub erstellt werden. Das geht auch über die GitBash, aber Ziel des PR ist eine Dokumentation und Kommunikation (Review mit 4-Augen-Prinzip)  
+    
+	
+    *Wie sieht der "git workflow" genau aus?*
+	
+    -> Habe dazu ein kurzes Video-Tutorial gemacht.  
+    
+	
+    *Was ist eine "gute Commit-Message"?*
+	
+    -> Verb - Nomen - #Ticketnummer
+	
+	
+    *Was ist eine "schlechte Commit-Message"?*
+    -> 
+    
+	
+    *Darf ich jemals direkt auf den Master-Branch pushen?*
+	
+    -> Nein!
+
```
- to much indentation
- add space after *

Braak-1991 replied:
my goal was to display the texts in cursive with * 
Oh well :D

nailend commented on protokolle/29-04-2020_protokoll.md:
```diff
@@ -0,0 +1,262 @@
+
+
+## Quick-Walkthrough: Protokolle aufbereiten 
+
+	* issue erstellen lohnt sich ab einer Bearbeungsdauer von 1 bis 2 Stunden
+	
+	* Hot-Fixes direkt drauf (Rechtschreibfehler, keine substanziellen Veränderungen, selbsterklärende Commit-Messages verwenden!)
+
```
remove indentation

nailend commented on protokolle/29-04-2020_protokoll.md:
```diff
@@ -0,0 +1,262 @@
+
+      * Fehlerquelle: Falsche DB ausgewählt und code somit auf falscher DB ausgeführt (Vorsicht!)
+
+    * *Wie führe ich auch einer Datenbank code aus?*
+
+      1. * Server auswählen
+
+      2. * Datenbank auswählen 
+
+      3. * Rechtskick: Querry-tool:
+
+         * Code einfügen und ausführen (F5)
+
+         * Querry wird immer mit Ausgabe "Query returned successfully in XXX msec." abgeschlossen
+
+         * Falls nicht, wurde nichts ausgeführt (All or nothing)
+
+         * **!Wichtig!** In der Regel muss ausgeführter CODE (rechts) im Browser (links) per Rechtsklick>Refresh neugeladen werden.
+
+      4. * Disconnected vom Server
+	      
+      5. * PostgresSQL 12: Properties ändern
+
+         * user == sonnja
+
+      6. * reconnecten mit Passwort : sonnja
+
+
+    * *FRAGE: Wie wurde die Datenbank "sonnja_db" hinzugefügt? Steht das in dem Skript 2?*
+
+    * -> Ja (Aber Prinzipiell Rechtsklick auf die Datenbank und neue Datenbank erzeugen)
+
+
+
+	* User Management mit Skript (von LH) erstellen:
+
+      * user Rolle erstellen
+
+      * admin Rolle erstellen
+
```
remove indentation

nailend commented on protokolle/29-04-2020_protokoll.md:
```diff
@@ -0,0 +1,262 @@
+   
+
+
+	* Fragen zu Git:
+
+      * -commit auf falschen branch a
+
+      * -wurde außerdem auf anderem branch b überholt
+
+      * -branch a ist aber noch nicht abgeschlossen
+
+      * -[Link](https://github.com/htw-pv3/literature/issues/11)
+
+	  * -guter Lösungsweg?
+
+        
+	* Sobald bei git nicht trivale Unterschiede auftauchen (z.B. eine Bearbeitung in der gleichen Zeit), tritt ein sognannter "Merge Conflict" auf.
+	* Das sieht zunächst schlimmer aus als es wirklich ist.
+	* Das macht kurz Angst, kann aber relativ einfach behoben werden.
+	* Ich schau es mir mal in Ruhe an und zeige eine einfache Lösung.
+	* Ich verwende das Programm Meld (http://meldmerge.org/) zum vergleichen von Dokumenten (Dif-Tool). Damit ist ein "manuelles Beheben" sehr einfach.
+
```
remove indentation

nailend commented on 29-04-2020_protokoll.md:
```diff
@@ -0,0 +1,262 @@
+# 2020-04-29 14:00 - 15:30 Wetterdatenaufbereitung mit PostgreSQL
```
remove this duplicate.

khelfen commented:
Checkout my [Cheatsheet for Markdown](https://github.com/htw-pv3/literature/blob/master/Cheatsheet_MD.md)!

Braak-1991 commented:
file should be up to date now
-removed the duplicate in the main folder
renamed the file in the protocol folder
removed the duplicate in the protocol folder
fixed some minor syntax issues

nailend commented on protokolle/2020-04-29_htw-pv3_postgresql.md:
```diff
@@ -0,0 +1,265 @@
+# Aufgaben
+
+
+   - [] Metadaten zu den CSV-Dateien "HTW-Wetter" und "Sonja-Messdaten" generieren [ALL]
+   
+   - [] die richtigen Datenbankbefehle für die Fehler -> Lücken finden und Lücken schließen [ALL]
```
- remove indentation
- remove bullets -
- add space between [] like [ ]

Braak-1991 replied:
Done

nailend commented on protokolle/2020-04-29_htw-pv3_postgresql.md:
```diff
@@ -0,0 +1,265 @@
+
+## Fragen zu Git:
+
+* -commit auf falschen branch a
+
+* -wurde außerdem auf anderem branch b überholt
+
+* -branch a ist aber noch nicht abgeschlossen
+
+* -[Link](https://github.com/htw-pv3/literature/issues/11)
+
+* -guter Lösungsweg?
+
+        
+* Sobald bei git nicht trivale Unterschiede auftauchen (z.B. eine Bearbeitung in der gleichen Zeit), tritt ein sognannter "Merge Conflict" auf.
+* Das sieht zunächst schlimmer aus als es wirklich ist.
+* Das macht kurz Angst, kann aber relativ einfach behoben werden.
+* Ich schau es mir mal in Ruhe an und zeige eine einfache Lösung.
+* Ich verwende das Programm Meld (http://meldmerge.org/) zum vergleichen von Dokumenten (Dif-Tool). Damit ist ein "manuelles Beheben" sehr einfach.
+
```
- rework structure. Not every line is a bullet point
- remove - after *
- find a better discription for the link like "Link to Issue"

Braak-1991 replied:
Also done :-)
